alts/src/main/proto/grpc/gcp/altscontext.proto

```diff
@@ -41,4 +47,4 @@ message AltsContext {
 
   // Additional attributes of the peer.
   map<string, string> peer_attributes = 7;
-}
+}
```

Comment:
nit: add new line pls. Same for two other proto files.

Reply:
Done.  It would be helpful if you could upstream this to the grpc-proto dir, and possibly to the C-core copy.

Reply:
Let me update grpc-proto dir.
C core will be updated by Yihua, in any case he need to update peer attribute field.